From d65ede3e874c79348efc85cf3a185c170046c4c8 Mon Sep 17 00:00:00 2001
From: Gustavo Zacarias <gustavo@zacarias.com.ar>
Date: Wed, 4 Dec 2013 13:31:28 -0300
Subject: [PATCH] Fix Makefile.am altivec logic

Besides SPE (FSL e500v? cores) there are other powerpc processors
that don't support altivec instructions so only enable them when it's
100% sure that the target has it.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>
Signed-off-by: Erik de Castro Lopo <erikd@mega-nerd.com>
---
 src/libFLAC/Makefile.am | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/libFLAC/Makefile.am b/src/libFLAC/Makefile.am
index 247e33c..258de40 100644
--- a/src/libFLAC/Makefile.am
+++ b/src/libFLAC/Makefile.am
@@ -47,8 +47,7 @@ CPUCFLAGS = -faltivec -force_cpusubtype_ALL -DFLAC__NO_ASM
 else
 # Linux-gcc for PPC does not have -force_cpusubtype_ALL, it is Darwin-specific
 CPUCFLAGS =
-if FLaC__CPU_PPC_SPE
-else
+if FLaC__USE_ALTIVEC
 CPUCFLAGS += -maltivec -mabi=altivec
 endif
 #@@@ PPC optimizations temporarily disabled
-- 
1.8.3.2

